File: Back/.Net/Modules/Bets/BetFriends.Bets.Infrastructure/Notifiers/NotifyBetCompleted.cs
using BetFriends.Bets.Application.Features.CompleteBet;
using BetFriends.Bets.Application.Features.CreateBet;

namespace BetFriends.Bets.Infrastructure.Notifiers;

internal class NotifyBetCompleted(IEnumerable<INotifier> notifiers) : INotifyBetCompleted
{
    private readonly IEnumerable<INotifier> notifiers = notifiers;

    public Task Notify(Notification notification)
    {
        return Task.CompletedTask;
    }
}

File: Back/NodeJS/modules/bets/src/infrastructure/notifiers/NotifyBetCompleted.ts
import { INotifyBetCompleted, Message } from "../../application/features/complete-bet/NotifyGamblersBetCompletedHandler";

export class NotifyBetCompleted implements INotifyBetCompleted {
    constructor(private readonly messages: Message[] = []) {}
    Notify(message: Message): Promise<void> {
        this.messages.push(message);
        return Promise.resolve();
    }

}